ftw/addressblock/geo/profiles/uninstall/jsregistry.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="portal_javascripts">
+
+  <javascript
+      id="++resource++ftw.addressblock.geo/addressblock.js"
```

Comment:
See https://github.com/4teamwork/ftw.addressblock/pull/32/files it's already gone